Feature/auth

yonatandt opened the pull request.

Shahar-Y committed:
added authenticator to services

Signed-off-by: shahar-Y <shya95@gmail.com>

yonatandt committed:
fix(auth): no need to send '/auth' again

Signed-off-by: Yonatan <yonatald@gmail.com>

omerzamir commented:
[This comment](https://github.com/meateam/kd-helm/issues/10#issuecomment-493765390) is relevant here too.

I'll add other comments and change requests later. But first merge your branch with develop, it's outdated.

yonatandt committed:
Merge branch 'develop' of https://github.com/meateam/kd-helm into feature/auth

omerzamir commented on authenticator/templates/ingress.yaml:
```diff
@@ -0,0 +1,25 @@
+    - host: {{ . | quote }}
+      http:
+        paths:
+          - path: /auth/?(.*)
```
Regex isn't necessary here.

omerzamir reviewed (changes requested):
Beside the comment on the regex, you should change the name of the authenticator repo to authentication-service. 
also, requirement.lock should be ignored with gitignore, it appears in the PR(that is the reason i asked for merge with develop). we should check that it's really ignored.

yonatandt committed:
fix(auth): Some fixes after pr review
- Change repo name of authentication-service
- Remove the regex from the ingress path of authenticator

Signed-off-by: Yonatan <yonatald@gmail.com>

omerzamir commented on authentication-service/values.yaml:
```diff
@@ -0,0 +1,63 @@
+replicaCount: 1
+
+image:
+  tag: "a15cb95"
```
wrong tag.
should be `tag: "3483326"`

omerzamir reviewed (changes requested):
see my comment on the tag

yonatandt committed:
update auth tag

Signed-off-by: Yonatan <yonatald@gmail.com>

yonatandt commented:
Why isn't the branch deleted after merge?
